python/ray/serve/tests/test_metrics.py

```diff
@@ -222,7 +222,7 @@ class A:
         async def __init__(self):
             pass
 
-        async def __call__(self):
+        async def __call__(self, *args):
```

Comment:
@GeneDer this test was not actually raising `RayActorError` because it was missing this argument and therefore raising a different error.

In the future please make sure to run each test you write manually and spot check the logs. It was obvious that something was wrong from them (spamming a bunch of unexpected errors).